src/transport/transport-utils.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include "serializer.h"
+#include "deserializer.h"
+
+namespace napa {
+namespace transport {
+
+    using namespace v8;
+
+    class TransportUtils {
+    public:
+        static std::shared_ptr<SerializedData>
+        SerializeValue(Isolate* isolate, Local<Value> value) {
+            bool ok;
+            Serializer serializer(isolate);
+            if (serializer.WriteValue(value).To(&ok)) {
+                return serializer.Release();
+            }
+            return nullptr;
+        }
+        
+        static MaybeLocal<Value>
+        DeserializeValue(Isolate* isolate, std::shared_ptr<SerializedData> data) {
```

Comment:
const std::shared_ptr<SerializedData>&

Reply:
made it std::shared_ptr<SerializedData>& without 'const'

v8 Deserializer require a non-const pointer of SerializedData


---
In reply to: [155906727](https://github.com/Microsoft/napajs/pull/137#discussion_r155906727) [](ancestors = 155906727)